src/components/PostPreview/PostPreviewFooter.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Skeleton, Text } from '@eleven-labs/design-system';
+import React from 'react';
+
+import { PostPreviewOptions, SeparatorCircle } from '@/components';
+
+const PostPreviewFooter = ({ isLoading, date, readingTime, authors }: Partial<PostPreviewOptions>): React.ReactNode => {
```

Comment:
Créer une interface dédié pour ce composant
```suggestion
const PostPreviewFooter: React.FC<PostPreviewFooterProps> = ({ isLoading, date, readingTime, authors }) => {
```